src/app/shared/components/delete-modal/delete-modal.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, EventEmitter, Output, ViewChild } from '@angular/core';
+import { NgbModal, NgbModalOptions, NgbModalRef } from '@ng-bootstrap/ng-bootstrap';
+import { TestListItem } from '../../interfaces/test-list-item';
+
+@Component({
+  selector: 'app-delete-modal',
+  standalone: true,
+  imports: [],
+  templateUrl: './delete-modal.component.html',
+  styleUrl: './delete-modal.component.css',
+})
+export class DeleteModalComponent {
+  @ViewChild('modal') modal!: NgbModal;
+  protected activeModal?: NgbModalRef;
+  @Output() confirmDeleteEvent = new EventEmitter<boolean>();
```

Comment:
This boolean is here to check if the user pressed on the delete all button or the button to delete a selected amount. So in the single emit I created, this can be true or false.